js/main.js

```diff
@@ -94,7 +100,7 @@ function creatingDOMTree(tagName, attributes, children = []) {
 }
 
 function creatingJournalEntry(newJournalEntry) {
-  var tree = creatingDOMTree('li', { class: 'entry-items' + ' ' + newJournalEntry.entryId }, [
+  var tree = creatingDOMTree('li', { class: 'data-entry-id' + ' ' + newJournalEntry.entryId }, [
```

Comment:
`data-entry-id` should be the name of the attribute, not a class name. Its value should be that entry's `id`.